### PR TITLE
Awesome books: plain JavaScript with objects

### PR DESCRIPTION
**Pull request by TadesseDev**
## Main changes we do.
---
#### changes by @Tadesse-Alemayehu 🖥 🖥 @oshanedesign 👬
- Create a collection that keeps a list of books ✔
- Create a function to add a new book to the collection ✔
- Create a function to remove a book from the collection ✔
- Display all books saved in the collection in the top part of the page ✔
- Preserve data in a browser memory. ✔

### CODE REVIEW

File: script/app.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+};
+
+const addBook = (title, author) => {
+  const newBook = {
+    id: String(bookCollection.length),
+    title,
+    author,
+  };
+  bookCollection.push(newBook);
+  const storeBooks = document.querySelector('#storeBooks');
```
Comment (gemmen29) [inline script/app.js:22]: [OPTIONAL] Kindly clear the input fields after pressing add button for a better user experience.

**Review by gemmen29 — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script/app.js (hunk 1 above)
Comment (TadesseDev) [reply script/app.js:22]: @gemmen29 Thank you for the thorough review provided. We will make the optional changes right away.

Comment (TadesseDev) [reply script/app.js:22]: @gemmen29 Optional changes have been implemented https://github.com/Tadesse-Alemayehu/Awesome-books/pull/1/commits/ae078dd12128336a98b2b96f3d729669306fe509. If there is anything we can improve on, please let us know.

Comment (gemmen29) [reply script/app.js:22]: Good job.

Comment (TadesseDev) [reply script/app.js:22]: Thank you ✌.

**Pull request merged**